VEDA/settings/utils.py

```diff
@@ -62,7 +62,7 @@ def get_logger_config(log_dir='/var/tmp',
         },
         'handlers': {
             'console': {
-                'level': 'DEBUG' if debug else 'INFO',
+                'level': 'INFO',  # TODO: Should be changed to -> 'DEBUG' if debug else 'INFO'
```

Comment:
@Qubad786 Can you look at the [logger settings](https://github.com/edx/edx-platform/blob/039afc2517c0fc70af8acb1a33b29c0809e503a4/openedx/core/lib/logsettings.py#L11) from edx-platform